### ISSUE TITLE
This project really needs a crates submission

### ISSUE THREAD

**Issue by chokosabe** [rj76/fcm-rust#11]
This project really needs a crates submission to help with versioning etc. Breaking changes were introduced in the last updates.

**Comment by chris13524** [comment]
What is a `crates submission`?

@rj76 as a side-note I think the version should be set to `0.x.x` instead of `1.x.x` so that breaking changes can continue to be made

**Comment by chokosabe** [comment]
I mean this isn't a documented crate. The only way in the docs to install this is:

fcm = { git = "https://github.com/rj76/fcm-rust.git" }

Ideally it'd be a cargo crate with a version number.

**Comment by rj76** [comment]
Yeah, when we're happy with the API and the docs, we should get it released there. @adnanjpg had some ideas about adding a pipeline to release to crates.io, that part is new to me...

**Comment by Ks89** [comment]
When will you publish this as crate?

**Comment by rj76** [comment]
I have no idea if this crate is used, as there are 6 other fcm v1 crates out there... But it could be published after bumping the version to 1.x.
